src/components/Footer.js

```diff
@@ -0,0 +1,21 @@
+import './footer.module.css';
+import socialLinks from '../data/socials.js';
+
+function Footer() {
+  return (
+    <footer>
+      <div></div>
+      <p>© Copyright 2023 John E. Mook</p>
+      <div>
+      {socialLinks.map((link) => {
+        return (
+          <a key={link.title} href={link.address}><img src={link.icon} alt={link.title} /></a>
```

Comment:
I have one tiny suggestion for not just these links, but with all the links, I think they could use the same target="_blank" treatment so external links open in another tab.

```suggestion
          <a key={link.title} href={link.address}><img src={link.icon} alt={link.title} rel="noopener noreferrer" target="_blank" /></a>
```